profiles/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ["vocabulary", "code", "translations"]
```

Comment:
pretty sure, it will not be able to resolve `"translations"`. you should do the same thing as in [venepaikka](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/harbors/schema.py#L14)

Reply:
also, the convention is to use tuples instead of lists for these things